packages/core/src/useCheckbox/useCheckbox.ts

```diff
@@ -100,6 +102,8 @@ export function useCheckbox<TValue = string>(
   _props: Reactivify<CheckboxProps<TValue>, 'schema'>,
   elementRef?: Ref<HTMLElement | undefined>,
 ) {
+  const form = inject(FormKey, null);
```

Comment:
Let's avoid doing that and instead expose the form on the `field` object so that we don't have to inject it multiple times.